src/app.ts

```diff
@@ -15,6 +15,10 @@ if (!server) {
   server = getServerDomain().replace(app, '');
 }
 
+const mock = parameters.has('mock');
+const mockParameter = parameters.get('mock');
+const mockUrl = mockParameter ? mockParameter : 'http://localhost:3000/mock.json';
+const miningUrl = mock ? mockUrl : 'tbd';
```

Comment:
Are you against the parameter `miningUrl` or why didn't you change it?
`const miningUrl = parameters.get('miningUrl') ?? 'http://localhost:3000/mock.json'`